Dockerfile

```diff
@@ -8,6 +8,13 @@ ENV CGO_ENABLED 0
 
 WORKDIR /go/src/github.com/minio/
 
+# add our user and group first with fixed uid and gid, this ensures
+# multiple updates to the minio image dont have permission issues.
+RUN addgroup -g 190 -S minio && adduser -u 190 -S -G minio minio
```

Comment:
why 190? you can perhaps set it to 1000

Reply:
idea is to add a system account, so a user on the machine does not get access to Minio data directory